controllers/nmstate_controller.go

```diff
@@ -189,6 +189,7 @@ func (r *NMStateReconciler) applyHandler(instance *nmstatev1beta1.NMState) error
 	data.Data["CAOverlapInterval"] = ""
 	data.Data["CertRotateInterval"] = ""
 	data.Data["CertOverlapInterval"] = ""
+	data.Data["WebhookReplicas"] = int32(2)
```

Comment:
Could you please move this line next to other Webhook variables? Either before `WebhookNodeSelector` or after `WebhookAffinity`.
Also, it would be nice if you could define a WEBHOOK_REPLICAS constant

Reply:
[Done.](https://github.com/nmstate/kubernetes-nmstate/compare/bbbb0a1989a87e288fc800d2f875f00d072f3c45..be645d50e4d2f2b3c16050ee4f82a9e5f33e73b2)